SWLOR.Game.Server/Conversation/BaseManagementTool.cs

```diff
@@ -62,19 +65,26 @@ public override PlayerDialog SetUp(NWPlayer player)
                 "75 degrees",
                 "90 degrees",
                 "180 degrees");
-            
-
+            DialogPage renamePage = new DialogPage("Type a name, Once you are done select confirm.",
+                "Confirm");
+            DialogPage confirmRenamePage = new DialogPage(
+    "<SET LATER>",
+    "Confirm Name Change"
+);
             dialog.AddPage("MainPage", mainPage);
             dialog.AddPage("PurchaseTerritoryPage", purchaseTerritoryPage);
             dialog.AddPage("StructureListPage", structureListPage);
             dialog.AddPage("ManageStructureDetailsPage", manageStructureDetailsPage);
             dialog.AddPage("RetrieveStructurePage", retrievePage);
             dialog.AddPage("RotatePage", rotatePage);
+            dialog.AddPage("RenamePage",renamePage);
+            dialog.AddPage("ConfirmRenamePage", confirmRenamePage);
             return dialog;
         }
 
         public override void Initialize()
         {
+            GetPC().SetLocalInt("LISTENING_FOR_DESCRIPTION", 1);
```

Comment:
Confused by this. I don't think it should be listening for a description on initialization of the conversation? This probably needs to be moved to a point when the player selects the rename option.